src/components/button/Button.js

```diff
@@ -50,6 +51,16 @@ const Button = (props) => {
                 ` : inactiveSolid
               break;
 
+            case "navigation":
+                styles += p.active ? `
+                    ${ButtonNavigationCSS}
+                ` : `
+                pointer-events: none;
+                color: ${colors.grey600};
+                font-size: 24px;
+                `
```

Comment:
I think for consistency, we should add these to a variable
You can refer to line 18 or 21 as an example :)